saleor/graphql/product/types.py

```diff
@@ -148,6 +147,12 @@ class Meta:
         interfaces = [relay.Node]
         model = models.ProductType
 
+    def resolve_products(self, info):
+        user = info.context.user
+        products = products_visible_to_user(
```

Comment:
Queryset could be returned in this line, there's no need for `products` variable.